install/install.go

```diff
@@ -1316,7 +1317,18 @@ func (k *K8sInstaller) generateConfigMap() (*corev1.ConfigMap, error) {
 		m.Data["enable-endpoint-routes"] = "true"
 		m.Data["auto-create-cilium-node-resource"] = "true"
 		m.Data["enable-local-node-route"] = "false"
-		m.Data["masquerade"] = "false"
+
+		ersion := strings.TrimPrefix(k.params.Version, "v")
+		if v, err := semver.Parse(ersion); err != nil {
```

Comment:
Other code in this package uses `github.com/cilium/cilium/pkg/versioncheck` for these kind of checks. Would that be appropriate here as well?

If not, we could at least use `github.com/blang/semver/v4` instead of `github.com/blang/semver` which is already vendored as a transitive dependency (once #358 is merged, that is).

Reply:
👍 for github.com/cilium/cilium/pkg/versioncheck